### PR TITLE
CTCP-2138: Mention PPNS in the API defintion

### PR DESCRIPTION
**Pull request by drnaylor-oc**
Not for merging until we've gotten our PPNS integration up and running.

### CODE REVIEW

File: resources/public/api/conf/2.0/application.raml [hunk 1]
```diff
@@ -102,7 +102,7 @@ traits:
 
         This request is also called an ‘Declaration Data’ E_DEC_DAT (IE015) message.
 
-        A successful response will contain a URI and a departure ID which you can use to retrieve messages related to the movement.
+        A successful response will contain a URI which you can use to retrieve the message status. A successful response can also include an optional box ID, which you can use to match this movement to any messages received from the Push Pull Notifications API.
```
Comment (fmcgrath) [inline resources/public/api/conf/2.0/application.raml:105]: Suggestion:
A successful response will contain a URI that you can use to retrieve the message status.

The issue here is grammar around usage of that/which. "That" is used in a defining sense whereas "which" is usually used in a commenting sense. If "which" is a better choice in a sentence, it must usually be prefixed by a comma. In this case, I think that "that" is a better fit for the sentence.

File: resources/public/api/conf/2.0/application.raml [hunk 2]
```diff
@@ -422,7 +423,7 @@ traits:
 
         This request is also called an ‘Arrival Notification’ E_ARR_NOT (IE007) message.
 
-        A successful response will contain a URI and an arrival ID which you can use to retrieve messages related to the movement.
+        A successful response will contain a URI which you can use to retrieve the message status. A successful response can also include an optional box ID, which you can use to match this movement to any messages received from the Push Pull Notifications API.
```
Comment (fmcgrath) [inline resources/public/api/conf/2.0/application.raml:427]: Same issue as above.

Suggestion:
A successful response will contain a URI that you can use to get the message status.

**Review by fmcgrath — COMMENTED** [review]
Two minor problems with a sentence that appears twice.

**Comment by drnaylor-oc** [comment]
@fmcgrath Updated as per your suggestions, thanks! Mind doing another quick sweep?

**Review by fmcgrath — APPROVED** [review]
All good.

**Comment by drnaylor-oc** [comment]
Thanks!

Will merge once we can deploy the push-notifications service to trader test, currently gremlins in the deploy system.

**Pull request merged**